all-the-icons.el

```diff
@@ -452,6 +452,8 @@
     ("\\.pp[st]x$"      all-the-icons-fileicon "powerpoint"             :face all-the-icons-red)
     ("\\.knt$"          all-the-icons-fileicon "powerpoint"             :face all-the-icons-cyan)
 
+    ("\\.xl\\(sx\\|sm\\|sb\\|tx\\|tm\\)$" all-the-icons-fileicon "excel" :face all-the-icons-dgreen)
```

Comment:
Please separate these extensions into different lines and minimize the use of complex regular expressions.